src/pages/TelaCadastroDadosAcesso.jsx

```diff
@@ -0,0 +1,195 @@
+import {
+    Text,
+    SafeAreaView,
+    StyleSheet,
+    TextInput,
+    View,
+    TouchableHighlight,
+} from 'react-native';
+import { useFonts } from 'expo-font';
+import AppLoading from 'expo-app-loading';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import routes from '../routes';
+import { useForm, Controller } from 'react-hook-form';
+
+export default function TelaCadastroDadosAcesso() {
+
+
+    const navigate = useNavigation().navigate
+    const dispatch = useDispatch()
+    const { control, handleSubmit, formState: { errors } } = useForm();
+    let [fontsLoaded] = useFonts({
+        'Barlow-Light': require("../assets/fonts/Barlow-Light.ttf"),
+    });
+    if (!fontsLoaded) {
+        return <AppLoading />
+    }
+    function buttonEntrar() {
+        navigate(routes.TELA_MENU)
+
+    }
+
+    return (
+        <SafeAreaView>
+            <View style={styles.div}>
+                <View style={styles.header}>
+                    <Text style={styles.dogtorText}>DOGTOR</Text>
+                    <Text style={{ fontWeight: 'bold', fontSize: 24 }}>Dados de Acesso</Text>
+                    <Text style={{ fontSize: 16 }}>Muito bom conhecer o dono desse pet. Precisamos das suas informações de acesso</Text>
+                </View>
+                <View>
+                    {<Controller
+                        control={control}
+                        name="email"
+                        rules={{ required: true }}
+                        render={({ field: { value, onChange } }) => (
+                            <TextInput
+                                style={{
+                                    height: 51,
+                                    width: 360,
+                                    marginTop: 12,
+                                    borderWidth: 1,
+                                    padding: 10,
+                                    borderRadius: 16,
+                                    borderColor: errors.email && 'red' || !errors.email && '#ACBBC3',
```

Comment:
aqui você pode utilizar um operador ternário!
```suggestion
                                    borderColor: errors.email ? 'red' : '#ACBBC3',
```

Acho uma boa substituir a string "#ACBBC3" para uma variável com o nome da cor também!

Reply:
feito